tools/autograd/derivatives.yaml

```diff
@@ -2852,7 +2852,7 @@
       self: unbind_backward(grads, dim)
       result: auto_linear
     AutogradNestedTensor:
-      self: unbind_backward_nested(grads, at::native::get_nested_tensor_impl(self)->get_nested_sizes(), dim, self.options())
+      self: "self.layout() == c10::kJagged ? unbind_backward_nested_jagged(grads, self, dim) : unbind_backward_nested(grads, at::native::get_nested_tensor_impl(self)->get_nested_sizes(), dim, self.options())"
```

Comment:
FWIW this means that the NST case also saves self now

Reply:
that is sad yes.. it's not ideal

Reply:
wait, it's already saving self due to `at::native::get_nested_tensor_impl(self)->get_nested_sizes()`